src/messages_pkg/igvc_msgs/msg/TrajectoryPoint.msg

```diff
@@ -0,0 +1,10 @@
+Header header
+
+# a [x,y,z] position in the global coordinate frame
+Point point
+
+# a scalar representing the longitudinal velocity
+float64 velocity
```

Comment:
May want to use a Vector3 for velocity for consistency.

Reply:
Updated TrajectoryPoint velocity to Vector3 in latest commit.

Reply:
Looks good!